README.md

```diff
@@ -60,7 +60,6 @@ import (
 ## In vscode
 There is launch configuration in `.vscode/launch.json` called `Launch demo suites`. 
 Running this configuration, you'll be asked for github token and then e2e-demos suite will run with default configuration.
-
```

Comment:
Why is this change part of this PR?

Reply:
Its part of the default file in https://github.com/redhat-appstudio/e2e-tests